dietpi/dietpi-software

```diff
@@ -10720,11 +10723,16 @@ _EOF_
 [Unit]
 Description=DietPi Gogs service
 DefaultDependencies=no
+#Optional
+#After=mysqld.service
```

Comment:
nice to have but optional if user decide to use sqllite...

Reply:
I would not expect end users to mess around with the systemd unit files. I would leave the `After=` active then, also because is does not hurt in any case.
Why I never found `mysqld.service` on my systems 🤔, just the `init.d` mysql, that at least listens to `systemctl restart mysql` (not `... mysqld`) and since Stretch additionally `mariadb.service`. For compatibility DietPi uses the mysql init.d service.

So, then to be fail safe, it would be `After=mysqld.service mysql.service mariadb.service`.
But this anyway has effect only, if controlled by systemd. DietPi-Services takes care by itself, that gogs is started after all kind of databases: https://github.com/Fourdee/DietPi/blob/master/dietpi/dietpi-services#L126

But again, it doesn't hurt to have these line active, then I suggest as above, with all possible mysql/mariadb service names.

Reply:
Added 👍